pkg/ebpf/tracee.go

```diff
@@ -328,14 +335,57 @@ func (t *Tracee) Init() error {
 		}
 	}
 
-	// the process-tree initialize must be before we start receiving events to ensure we already have the needed data for the events that use that tree
+	// Initialize process tree before receiving events (will use it)
+
 	t.procInfo, err = procinfo.NewProcessInfo()
 	if err != nil {
 		t.Close()
 		return fmt.Errorf("error creating process tree: %v", err)
 	}
 
-	t.containers, err = containers.New(t.config.Sockets, "containers_map", t.config.Debug)
+	// Initialize cgroup filesystems
+
+	var defaultCgroup cgroup.Cgroup
+
+	isCgroupV1, err := cgroup.IsCgroupV1Default()
+	if err != nil {
+		return err
+	}
+	t.cgroupv1 = nil
```

Comment:
No need, this should start as nil by default.

Reply:
I wanted to make sure (when testing =D).